File: data/submission/265973/A/43154080TuanDV_80.java
def load_input():
    line_1 = input().split()
    N, K = int(line_1[0]), int(line_1[1])
    arr = [int(i) for i in input().split()]
    return N, K, arr

def solution(K, N, arr):
    count = 0
    for i in range(N):
        tong = 0
        for j in range(i, N):
            tong += arr[j]
            if tong % K == 0:
                count += 1
    print(count)

if __name__ == '__main__':
    N, K, arr = load_input()
    solution(K, N, arr)

File: server/data/submission/265973/A/43154080TuanDV_80.cc
#include<stdio.h>
#include<stdlib.h>
#include<math.h>
#define Max 100

int n;
int k;
int arr[Max + 1]; 
void input(){
    scanf("%d %d", &n, &k);
    for(int i = 0; i < n; i++){
        scanf("%d", &arr[i]);
    }
}
void solution(){
    int count = 0;
    for(int i = 0; i< n; i++){
        int tong = 0;
        for(int j = i; j < n; j++){
            tong += arr[j];
            if (tong % k == 0){
                count ++;
            }
        }
    }
    printf("%d", count);
}
int main(){
    input();
    solution();
    return 0;
}